### PR TITLE
suppress recursive path

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2346
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/420
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/273
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/781
https://github.com/open-orchestra/open-orchestra-model-interface/pull/315
https://github.com/open-orchestra/open-orchestra/pull/1233

### CODE REVIEW

File: Media/Repository/FolderRepositoryInterface.php [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
     /**
-     * @param string $parentId
+     * @param string $path
      * @param string $siteId
      *
      * @throws \Exception
      *
      * @return Collection
      */
-    public function findByParentAndSite($parentId, $siteId);
```
Comment (alavieille) [inline Media/Repository/FolderRepositoryInterface.php]: not remove the old method. mark it like deprecated

**Comment by itkg-nanne** [comment]
update done @alavieille

File: Media/Repository/FolderRepositoryInterface.php [hunk 2]
```diff
@@ -18,6 +18,17 @@
     public function find($id);
 
     /**
+     * @param string $parentId
+     * @param string $siteId
+     *
+     * @throws \Exception
+     *
+     * @deprecated
+     * @return Collection
```
Comment (alavieille) [inline Media/Repository/FolderRepositoryInterface.php]: return an array

File: MediaModelBundle/Repository/FolderRepository.php [hunk 3]
```diff
@@ -14,6 +14,27 @@
...
 {
+
+    /**
+     * @param string $parentId
+     * @param string $siteId
+     *
+     * @throws \Exception
+     *
+     * @deprecated
+     * @return Collection
```
Comment (alavieille) [inline MediaModelBundle/Repository/FolderRepository.php]: return an array

**Comment by alavieille** [comment]
resolve https://insight.sensiolabs.com/projects/b30b397c-3dc1-4a97-a0ee-7ef63a015795/analyses/511?status=commit

**Pull request merged**